web/frontend/src/boot/i18n.ts

```diff
@@ -3,14 +3,12 @@ import { createI18n } from 'vue-i18n';
 import messages from '../i18n';
 
```

Comment:
The changes in this file are to help with allowing i18n from arbitrary functions, not just directly inside Vue components.